src/views/teachers/faq/faq.jsx

```diff
@@ -30,55 +30,151 @@ const TeacherFaq = props => (
                     <dt><FormattedMessage id="teacherfaq.teacherSignUpTitle" /></dt>
                     <dd><FormattedHTMLMessage id="teacherfaq.teacherSignUpBody" /></dd>
                     <dt><FormattedMessage id="teacherfaq.teacherWaitTitle" /></dt>
-                    <dd><FormattedHTMLMessage id="teacherfaq.teacherWaitBody" /></dd>
+                    <dd><FormattedMessage id="teacherfaq.teacherWaitBody" /></dd>
+                    <dt><FormattedMessage id="teacherfaq.classMultipleTeachersTitle" /></dt>
+                    <dd><FormattedMessage id="teacherfaq.classMultipleTeachersBody" /></dd>
+                    <dt><FormattedMessage id="teacherfaq.convertToTeacherTitle" /></dt>
+                    <dd>
+                        <FormattedMessage
+                            id="teacherfaq.convertToTeacherList"
+                            values={{
+                                helpEmail: (
+                                    <a href="mailto:help@scratch.mit.edu">
+                                        help@scratch.mit.edu
+                                    </a>
+                                )
+                            }}
+                        />
+                    </dd>
+                    <ul>
+                        <li><FormattedMessage id="teacherfaq.convertToTeacherUsername" /></li>
+                        <li><FormattedMessage id="teacherfaq.convertToTeacherEmail" /></li>
+                        <li><FormattedMessage id="teacherfaq.convertToTeacherBirth" /></li>
+                    </ul>
+
                     <dt><FormattedMessage id="teacherfaq.teacherPersonalTitle" /></dt>
-                    <dd><FormattedHTMLMessage id="teacherfaq.teacherPersonalBody" /></dd>
+                    <dd><FormattedMessage id="teacherfaq.teacherPersonalBody" /></dd>
                     <dt><FormattedMessage id="teacherfaq.teacherGoogleTitle" /></dt>
-                    <dd><FormattedHTMLMessage id="teacherfaq.teacherGoogleBody" /></dd>
+                    <dd><FormattedMessage id="teacherfaq.teacherGoogleBody" /></dd>
+                    <dt><FormattedMessage id="teacherfaq.studentDiscussTitle" /></dt>
+                    <dd>
+                        <FormattedMessage
```

Comment:
I think you need to keep this as a FormattedHTMLMessage (at least for now, and for a while) because existing translations are going to have HTML in them.